nvflare/app_common/executors/statistics/statistics_executor.py

```diff
@@ -123,6 +124,9 @@ def _client_exec(self, task_name: str, shareable: Shareable, fl_ctx: FLContext,
             ds_features = self.get_numeric_features()
             metric_task = shareable.get(StC.METRIC_TASK_KEY)
             target_metrics: List[MetricConfig] = fobs.loads(shareable.get(StC.STATS_TARGET_METRICS))
+            if StC.STATS_FAILURE_COUNT not in target_metrics:
+                target_metrics.append(MetricConfig(StC.STATS_FAILURE_COUNT, {}))
+
```

Comment:
not related to this PR, but I think if "initialization" failed in this class, we should just self.system_panic or something equivalent to stop this executor on client side.

we should not record the `init` failure and respond in `execute`.

Reply:
Server should have died when all clients call system panics => yes, I agree, we lack this mechanism now